src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -134,21 +134,11 @@ private void Initialize()
         {
             if (string.IsNullOrWhiteSpace(Project))
             {
-                string directory = Directory.GetCurrentDirectory();
-                string[] projectFiles = Directory.GetFiles(directory, "*.*proj");
-
-                if (projectFiles.Length == 0)
-                {
-                    var project = "--project";
-
-                    throw new GracefulException(LocalizableStrings.RunCommandExceptionNoProjects, directory, project);
-                }
-                else if (projectFiles.Length > 1)
-                {
-                    throw new GracefulException(LocalizableStrings.RunCommandExceptionMultipleProjects, directory);
-                }
-
-                Project = projectFiles[0];
+                Project = FindSingleProjectInDirectory(Directory.GetCurrentDirectory());
```

Comment:
Because I'm hot a huge fan of repurposing properties a lot.. but I'll change it that way